core/vm/jump_table.go

```diff
@@ -1106,6 +1106,21 @@ func newFrontierInstructionSet() JumpTable {
 			valid:      true,
 			writes:     true,
 		},
+		JUMPSUB: {
```

Comment:
Don't add it to Frontier -- look into eips.go, and define it as a later add-on to a new instructionset

Reply:
I'm lost here.

Reply:
See e.g. https://github.com/ethereum/go-ethereum/blob/master/core/vm/eips.go#L47 -- where we add new opcodes for EIP 1884

Reply:
Doing it that way makes it possible to execute state-tests with e.g. `Byzantium+2315` as identifier

Reply:
Got it, I think.  Thanks,